src/legacy/core_plugins/kibana/public/discover/directives/histogram.tsx

```diff
@@ -48,18 +48,43 @@ import { i18n } from '@kbn/i18n';
 import chrome from 'ui/chrome';
 // @ts-ignore: path dynamic for kibana
 import { timezoneProvider } from 'ui/vis/lib/timezone';
+import { EuiChartThemeType } from '@elastic/eui/src/themes/charts/themes';
+import { EUI_CHARTS_THEME_LIGHT } from '@elastic/eui/dist/eui_charts_theme';
+import { Subscription } from 'rxjs';
 
 export interface DiscoverHistogramProps {
   chartData: any;
   timefilterUpdateHandler: (ranges: { from: number; to: number }) => void;
 }
 
-export class DiscoverHistogram extends Component<DiscoverHistogramProps> {
+interface DiscoverHistogramState {
+  chartsTheme: EuiChartThemeType['theme'];
+}
+
+export class DiscoverHistogram extends Component<DiscoverHistogramProps, DiscoverHistogramState> {
   public static propTypes = {
     chartData: PropTypes.object,
     timefilterUpdateHandler: PropTypes.func,
   };
 
+  private subscription?: Subscription;
+  public state = {
+    chartsTheme: EUI_CHARTS_THEME_LIGHT.theme,
```

Comment:
@joshdover QQ: I feel like this part defeats part of the purpose of the plugin in which, they Kibana shouldn't even need to know where this chart theme is and what the default should be.

Maybe this should be a constant that the plugin provides?

Reply:
Also, thank you for all your help so far.

It looks like CI failed on a jest test on this import:
`import { useObservable } from 'src/plugins/kibana_react/public';`

Reply:
I've added a React hook just for using the charts theme that includes the default value. For cases like this where we can't use hooks yet (because this component hasn't been converted), I've added a `getChartsThemeDefault()` method to the plugin.